python/kserve/kserve/ray_model_server.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 The KServe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+
+try:
+    from ray import serve as rayserve
+    from ray.serve.deployment import Deployment
+    from .ray_model import RayServeModel
+except ImportError:
+    raise ImportError("Missing optional dependency 'ray.serve'. "
+                      "To use KServe with Ray Serve run pip install kserve[ray-serve]")
+
+
+from .model_server import ModelServer
+
+
+class RayModelServer(ModelServer):
+    """Ray ModelServer
+    """
+
+    def start(self, models: Dict[str, Deployment]) -> None:
```

Comment:
noob question: Is it a good idea to overload a method with different parameter types in a Python subclass?